packages/shared/src/components/notifications/EnableNotification.tsx

```diff
@@ -57,6 +57,133 @@ const sourceToButtonText: Partial<Record<NotificationPromptSource, string>> = {
   [NotificationPromptSource.SourceSubscribe]: 'Enable',
 };
 
+interface EnableNotificationAlertCTA {
```

Comment:
Following the [discussion here](https://dailydotdev.slack.com/archives/C07ATUPBPJ9/p1721655993518569), I think we can revert the changes to this file if the only reason was to support the banner.